build-logic/basics/src/main/kotlin/gradlebuild/basics/BuildParams.kt

```diff
@@ -55,6 +55,12 @@ import gradlebuild.basics.BuildParams.TEST_SPLIT_ONLY_TEST_GRADLE_VERSION
 import org.gradle.api.JavaVersion
 import org.gradle.api.Project
 import org.gradle.api.provider.Provider
+import org.jetbrains.kotlin.util.capitalizeDecapitalize.toUpperCaseAsciiOnly
+
+
+enum class FlakyTestStrategy {
+    INCLUDE, EXCLUDE, QUARANTINE
```

Comment:
How about this:

```suggestion
    ALL, NO_FLAKY, ONLY_FLAKY
```

Maybe this should then also be the `RunTestsStrategy`, WDYT?